src/components/organisms/Top.tsx

```diff
@@ -12,20 +12,24 @@ export type WeatherData = {
   weather_icon: string;
 };
 
-export type LivaData = {
+export type LiveData = {
   temperature_celsius: number;
   humidity: number;
+  alarm: boolean;
 };
 
 type Props = {
   deviceId: string | null;
 };
 
+const device_id = "a7382f5c-3326-4cf8-b717-549affe1c2eb";
```

Comment:
Get device id using GET `/user/devices`

Reply:
Updated, we can actually use props being passed in from index.tsx where the deviceId is retrieved.